lib/xcov/model/report.rb

```diff
@@ -81,5 +88,19 @@ def self.excluded_targets
       excluded_targets
     end
 
+    def self.included_targets
```

Comment:
What do you think if we refactor this like so?
```ruby
def self.included_targets
  include_targets = Xcov.config[:include_targets]
  return [] if include_targets.nil?

  if include_targets.is_a?(Array)
    include_targets
  else
    include_targets.split(/\s*,\s*/)
  end
end
```
I find this shorter and easier to reason about due to fewer nested levels.
We could also use ternary operator, but I think `if/else` makes it easier to understand.